src/passes/addressHandler/addressArgumentPusher.ts

```diff
@@ -1,37 +0,0 @@
-import {
```

Comment:
Why was this removed?

Reply:
Yeah, doing the transform for cross contract calling in the AST is a huge pain to do in a way that works nicely with other passes and doesn't have to be constantly designed around anytime we process functions. The addressHandler and its constituent passes have been deprecated for a while because of this, they were only kept in to throw a NotSupportedYet when they would have pushed an argument. Now that such calls are supported they're not needed.